docs/source/version-5-integrating.md

```diff
@@ -27,3 +27,59 @@ With Coral setup and running you can embed the comment stream with this sample e
 ```
 
 > **NOTE:** Replace the value of `{% raw %}{{ CORAL_DOMAIN_NAME }}{% endraw %}` with the location of your running instance of Coral.
+
+
+## Story Creation
+
+Lazy `Story` Creation enables stories to be automatically created when they are published from your CMS. Triggering the embed script above renders the comment stream iFrame on your page. By default that script dynamically generates `Stories` in Coral for seamless integration.
```

Comment:
> `Story` and `Stories`

Shouldn't be wrapped in inline code blocks here.

Reply:
This was done to highlight that `Story` and `Stories` are objects/models in Coral.